source/refined-github.css

```diff
@@ -180,3 +180,8 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 .octicon-star-fill {
 	color: var(--button-star-iconColor);
 }
+
+/* Untruncate long job name #8372 */
```

Comment:
Every feature must have a test URL, follow the format of other selectors in this file